### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by xinzhongyouhai** [open-webui/open-webui#8058] [Q&A]
0.4.8 版本~  管理员可以选择模型~而非管理员账户都无法选择模型~~~

**Comment by stollor** [comment]
我也遇到了这个问题,  开通了默认用户组访问模型列表的全选,但没有生效

**Comment by stollor** [comment]
我解决这个问题了,需要在 设置-模型 界面中,把模型的可见性改为public

**Reply by cs3306** [reply]
感谢大佬，脑细胞差点烧干了。。。

**Reply by freeznight** [reply]
感谢，救我一命！

**Reply by mkblbj** [reply]
感谢！！！

**Reply by NEPDream** [reply]
> 我解决这个问题了,需要在 设置-模型 界面中,把模型的可见性改为public

不能批量设置，让人有点头大
